<launch>
    <node name="control_node" pkg="control" type="control" output="screen">
        <rosparam command="load" file="$(find control)/config/control.yaml" /> <!--Load parameters from config files-->
        <rosparam command="load" file="$(find control)/config/control_para.yaml" />
        <param name="lqr_para_filename" value="$(find control)/config/lqr_para/lqr_para3.txt" />
    </node>
</launch>
